state-chain/pallets/cf-ingress-egress/src/tests.rs

```diff
@@ -892,6 +893,71 @@ fn deposits_below_minimum_are_rejected() {
 	});
 }
 
+#[test]
+fn deposits_tx_fee_exceeding_deposit_amount_rejected() {
+	let eth = eth::Asset::Eth;
+
+	new_test_ext().execute_with(|| {
+		// Set Eth fees to some arbitrary value, high enough for our test swap
+		TrackedDataProvider::<Ethereum>::set_tracked_data(cf_chains::eth::EthereumTrackedData {
+			base_fee: 100,
+			priority_fee: 0,
+		});
+
+		let (_id, address, ..) =
+			IngressEgress::request_liquidity_deposit_address(ALICE, eth).unwrap();
+
```

Comment:
`let deposit_address = address.try_into().unwrap();`

save some duplication of the try_into().unwrap()